File: AndroidApps/TicTacToeApp/app/src/main/java/com/example/tictactoeapp/MainActivity.java
package com.example.tictactoeapp;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    /*
    * Author: Kwangjin Baek
    * Date: 2022. Feb. 7.
    * Description: Tic Tac Toe game application
    * */

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void onStartClick(View view) {
        Intent intent = new Intent(this, GameBoardActivity.class);
        startActivity(intent);
    }
}

File: AndroidApps/PracticeForm/app/src/main/java/com/jinco/practiceform/MainActivity.kt
package com.jinco.practiceform

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.ArrayAdapter
import com.jinco.practiceform.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    lateinit var binding:ActivityMainBinding
    lateinit var user: User
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        val view = binding.root
        setContentView(view)
        setupButton()
        setSpinner()
    }

    private fun setupButton() {
        binding.btnCreate.setOnClickListener {
            createAccount()
        }
    }

    private fun createAccount() {
        user = User(
            binding.etFirstName.text.toString(),
            binding.etLastName.text.toString(),
            binding.spTitle.selectedItem.toString(),
            binding.etPassword.text.toString(),
            binding.etEmail.text.toString(),
            binding.etPhone.text.toString()
        )
        val intent = Intent(this, SummaryActivity::class.java)
        intent.putExtra("user", user)
        startActivity(intent)
    }

    private fun setSpinner() {
        val titleSpinner = arrayOf("1", "2")
        val titleAdapter = ArrayAdapter(this, android.R.layout.simple_spinner_dropdown_item, titleSpinner)
        binding.spTitle.adapter = titleAdapter
    }
}